app/build.gradle

```
@@ -45,19 +51,19 @@ android {
 
 dependencies {
     implementation fileTree(dir: "libs", include: ["*.jar"])
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    implementation "org.jetbrains.kotlin:kotlin-stdlib:1.7.10"
```

```suggestion
    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
```
leave as variable value